.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -42,3 +42,7 @@ Steps to reproduce the behavior:
 **Additional context**
 
 <!-- Add any other context about the problem here. -->
+
+**Want to contribute?**
+
+We love contributions from the Ethereum community! Comment on this issue below if you're interested in helping out with a PR.
```

Comment:
Not clear to me what the "issue below" would be? Do you mean they should comment on the issue they're creating to signal interest?

If so, maybe something like

"We love contributions from the Ethereum community! Remember to comment on your issue if you're interested in helping out with a PR".

Reply:
Ya @ryancreatescopy this came from a conversation in retro last week. Main goal here is to have this prompt not for the creator of the issue but for folks who come across an open issue.